pyomo/solvers/plugins/solvers/CBCplugin.py

```diff
@@ -563,7 +563,7 @@ def process_logfile(self):
                 # https://projects.coin-or.org/Cbc/browser/trunk/Cbc/src/CbcSolver.cpp?rev=2497#L7918
                 elif tokens[0] == 'Gap:':
                     # This is relative and only to 2 decimal places - could calculate explicitly using lower bound
-                    gap = float(tokens[1])
+                    gap = float(tokens[1]) if tokens[1] != '1.#J' else float('inf')
```

Comment:
Is this the only time that CBC can return `1.#J`?  I wonder if a better solution would be a standard function that was something like:
```python
def _float(x):
    if x == '1.#J':
        return float('inf')
    return float(x)
```
and then use that function in lieu of `float(x)` throughout the parser?

Reply:
Good idea, I'll include that.